doc/div-alg.xml

```diff
@@ -348,7 +350,11 @@ gap> SplittingDegreeAtP(F,120,5); SplittingDegreeAtP(K,120,5); last2/last;
   <Ref Attr="ReducingCyclotomicAlgebra"/> is another subroutine used by the
   <Ref Attr="GlobalSplittingOfCyclotomicAlgebra"/> that either returns an isomorphic
   cyclotomic algebra whose associated Galois group has fewer generators than
-  the input, or returns fail.
+  the input, or returns fail.  <Ref Attr="CyclotomicExtensionGenerator"/> is a
+  subroutine called by <Ref Attr="ReducingCyclotomicAlgebra"/> which returns a
+  positive integer <C>m</C> if the field <C>K</C> is the cyclotomic extension of
```

Comment:
A proper GAPDoc syntax would be to use
```
positive integer <M>m</M> if the field <A>K</A> is the cyclotomic extension of 
```
where `<A>` is for argument and `<M>` is for maths. But this is inconsistent everywhere, and it's not easy for your to produce documentation, so I will take care of applying this myself - added TODO to #58.